src/common/config/decrypt.tsx

```diff
@@ -64,28 +64,40 @@ export const decryptWalletOrSigner = async (
   progressCallback: (progress: number) => void
 ): Promise<Wallet | ConnectedSigner> => {
   const provider = config.network === "local" ? new providers.JsonRpcProvider() : getDefaultProvider(config.network);
-
   if (isWalletOption(config.wallet)) {
     const decryptedWallet = await Wallet.fromEncryptedJson(config.wallet, password, progressCallback);
```

Comment:
since now the shape is standardised to check for wallet type, this condition if of `isWalletOption` should no longer be needed?

Reply:
There is still a need to check if the wallet is string for backward compatibility

Reply:
right ok